src/Controller/ContentController.php

```diff
@@ -48,7 +48,7 @@ class ContentController
      * @param ViewHandlerInterface $viewHandler     Optional view handler
      *                                              instance
      */
-    public function __construct(EngineInterface $templating, $defaultTemplate, ViewHandlerInterface $viewHandler = null)
```

Comment:
shouldn't you set a kind of `ignore-if-not-exists`? in service definition? i did it here: https://github.com/egeloen/IvoryCKEditorBundle/pull/340/files#diff-b96d5122f1d04a59e13c03c7a4c39167R14 as the template is a parameter, right?

Reply:
this comment should go on the changed line :-(

Reply:
the parameter is defined but defaults to null. it looks like symfony in that case tries to autowire and fails, except when we explicitly state in the constructor that the value may be empty.

Reply:
ok, now i got it :-) and we also have a null, by defining the parameter. But should work this way.